velox/exec/Task.cpp

```diff
@@ -2116,10 +2116,10 @@ std::string Task::toShortJsonString() const {
   obj["numThreads"] = numThreads_;
   obj["terminateRequested_"] = std::to_string(terminateRequested_);
   obj["pauseRequested_"] = std::to_string(pauseRequested_);
-  return folly::toPrettyJson(obj);
+  return obj;
```

Comment:
Why you are changing this, would you fix property names to remove trailing underscore?

pauseRequested_ -> pauseRequested

and use boolean types instead of strings:

```
obj["pauseRequested"] = pauseRequested_;
```